climpred/metrics.py

```diff
@@ -92,23 +173,19 @@ def _less(forecast, reference, dim='svd', comparison=None):
     """
     Logarithmic Ensemble Spread Score.
 
-    Formula
-    -------
-    .. math:: LESS = ln(\frac{\sigma^2_F}{\sigma^2_R})
-
-    Reference
-    ---------
-    * Kadow, Christopher, Sebastian Illing, Oliver Kunst, Henning W. Rust,
-      Holger Pohlmann, Wolfgang A. Müller, and Ulrich Cubasch. “Evaluation of
-      Forecasts by Accuracy and Spread in the MiKlip Decadal Climate Prediction
-      System.” Meteorologische Zeitschrift, December 21, 2016, 631–43.
-      https://doi.org/10/f9jrhw.
-
-    Range
-    -----
-    pos: under-disperive
-    neg: over-disperive
-    perfect: 0
+    .. math:: LESS = ln(\\frac{\sigma^2_F}{\sigma^2_R})
```

Comment:
I think this should be "f" and "o" for consistency with the rest of the documentation